qutebrowser/config/configdata.yml

```diff
@@ -1012,6 +1012,32 @@ input.spatial_navigation:
     Right key, heuristics determine whether there is an element he might be
     trying to reach towards the right and which element he probably wants.
 
+# TODO set to false before merging!
+input.blur_on_load.enabled:
+  default: true
+  type: Bool
+  desc: >-
+    Clear focus on page during page load.
+
+    This prevents the page from stealing focus on page load, similar to how
+    js vim plugins preserve their own focus.
+
+input.blur_on_load.delay:
+  default: 0
+  type: Int
+  desc: >-
+    Delay (in milliseconds) after load finished before sending a second blur
+    event.
+
+    Only useful if input.blur_on_load.enabled is true.
+
+# TODO set to false before merging!
```

Comment:
Given that people already complained in `#qutebrowser` with the accidental focus leave, I think it'd be good to default to false.